src/SonarQube.Bootstrapper/Program.cs

```diff
@@ -89,5 +102,30 @@ private static ILegacyTeamBuildFactory GetLegacyTeamBuildFactory()
             return new NotSupportedLegacyTeamBuildFactory();
 #endif
         }
+
+        [Conditional("DEBUG")]
```

Comment:
Dumps out details about all of the loaded assemblies.
I've done this at the end of the process so we log all of the assemblies that have been used.
I added this mainly to help with testing the NET Core support. Before, I could only tell of the process completed successfully or not. With the logging I can check see if the expected assemblies were used and where they were loaded from.

Currently the code is only included in debug builds i.e. it won't ship to customers. Do you think it would be useful to ship it in release mode too?

Reply:
Do you think it will be useful after we fix initial issues? If you think it would, let's add it. That data would appear only in debug log, so it won't boher users if we add it.